src/test/java/com/artipie/rpm/http/RpmSliceITCase.java

```diff
@@ -61,6 +63,21 @@
      */
     private static final Vertx VERTX = Vertx.vertx();
 
+    /**
+     * Installed packages verifier.
+     */
+    private static final ListOf<String> INSTALLED = new ListOf<>(
+        "Installed", "aspell-12:0.60.6.1-9.el7.x86_64",
+        "time-1.7-45.el7.x86_64", "Complete!"
+    );
+
+    /**
+     * Packaged list verifier.
+     */
+    private static final ListOf<String> LIST = new ListOf<>(
```

Comment:
@olenagerasimova I'd suggest to change this field's name to more informative one, such as `AVAILABLE`. This way difference with `INSTALLED` will be obvious